### ISSUE TITLE
Incorrect model architecture in decode.py 

### ISSUE THREAD

**Issue by Chandrahas-B** [samarthagali/audiolite#1]
I noticed that you are loading a Conv1d layer and passing the decoder outputs in decode.py

Lines 54 and 55:

`final = nn.Conv1d(in_channels=1,out_channels=1,kernel_size=1,padding=0,stride=1).to(device)
 final.load_state_dict(torch.load("model_weights/final.pt"), strict=False)`

We had finalised our architecture to not have this layer, right?

Why is it still present in the validation scripts?

**Comment by samarthagali** [comment]
Skip connections has these.

**Comment by samarthagali** [comment]
Without skip connections doesn't have.

**Comment by samarthagali** [comment]
Please recheck if you've trained anything else tell.

**Comment by Chandrahas-B** [comment]
From my understanding, we have removed the final layer from the architecture.

Please finalise with @bhavithpbrao and resolve it.

**Comment by samarthagali** [comment]
Not for the skip connections model
 Even training used it.
